Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -668,15 +667,62 @@ public static int Remove(byte* pathname)
         return 0;
     }
 
-    private static unsafe void* GetLastStream()
+    internal static StreamHandle? GetStream(IntPtr handle)
```

Comment:
Can we accept `void* filePtr` parameter here and perform cast inside? I think otherwise it's unnescessary noise to convert everywhere.